frontend/mgramseva/lib/screeens/landing_page/landing_page_new.dart

```diff
@@ -29,82 +27,6 @@ class _LandingPageNewState extends State<LandingPageNew> {
       super.dispose();
     }*/
 
-  var state = [
-    'Andhra Pradesh',
-    'Arunachal Pradesh',
-    'Assam',
-    'Bihar',
-    'Chhattisgarh',
-    'Goa',
-    'Gujarat',
-    'Haryana',
-    'Himachal Pradesh',
-    'Jharkhand',
-    'Karnataka',
-    'Kerala',
-    'Madhya Pradesh',
-    'Maharashtra',
-    'Manipur',
-    'Meghalaya',
-    'Mizoram',
-    'Nagaland',
-    'Odisha',
-    'Punjab',
-    'Rajasthan',
-    'Sikkim',
-    'Tamil Nadu',
-    'Telangana',
-    'Tripura',
-    'Uttar Pradesh',
-    'Uttarakhand',
-    'West Bengal',
-  ];
-
-/*  final List<ToggleItem> items =
-  [ToggleItem('KARNATAKA', false),ToggleItem('ASSAM', false),ToggleItem('GUJRAT', false)];
-  //['KARNATAKA', 'ASSAM', 'GUJRAT', 'STATE', 'STATE'];*/
-
-  void shuffeling() {
-    for (int i = 0; i < state.length; i++) {
-      var newVar = typedText;
-      typedText = typedText.toLowerCase();
-
-      if (state.elementAt(i).toLowerCase().contains(typedText)) {
-        typedText = state.elementAt(i);
-
-        state[i] = state[0];
-        state[0] = typedText;
-        //items.insert(i, typedText);
-        //items[i].text = typedText;
-        break;
-      }
-    }
-    //items.shuffle();
-    setState(() {});
-  }
-
-  void shuffleList() {
-    for (int i = 0; i < state.length; i++) {
-      if (state.elementAt(i).contains(typedText)) {
```

Comment:
why all this functions are removing ? @shubhamdevgupta

Reply:
those are i added when i run in mobile application so now i will read state from asserts/state.json